A32NX/html_ui/Pages/A32NX_Utils/NXSystemMessages.js

```diff
@@ -1,46 +1,75 @@
+class McduMessage {
+    constructor(_text = "", _isAmber = false, _isTypeTwo = false, _subDefault = "", _subAfter = true) {
+        this.cText = _text;
+        this.isAmber = _isAmber;
+        this.isTypeTwo = _isTypeTwo;
+        this.sText = _subDefault;
+        this.isSAfterC = _subAfter;
+    }
+
+    set text(_t) {
+        console.error("it's not allowed to edit the text component of a message!");
+    }
```

Comment:
I would just omit this, which should forbid it. Rather, add a comment to `get text()` to tell future developers looking at this file why this cannot be set.

```suggestion
```